--- a/buddybird-app/edge.py
+++ b/buddybird-app/edge.py
@@ -349,7 +349,7 @@
             name="Managed-CachingDisabled"
         )
         all_viewer = aws.cloudfront.get_origin_request_policy_output(
-            name="Managed-AllViewer"
+            name="Managed-AllViewerAndCloudFrontHeaders-2022-06"
         )
         host_only = aws.cloudfront.get_origin_request_policy_output(
             name="Managed-HostHeaderOnly"
